moto/events/utils.py

```diff
@@ -1,3 +1,25 @@
+from typing import TYPE_CHECKING, List, TypedDict, Union
+
+if TYPE_CHECKING:
+    from typing_extentions import Any, Dict, Required
+
+
+# NOTE: Typing is based on the following document https://docs.aws.amazon.com/eventbridge/latest/userguide/eb-event-patterns.html
+EventMessageType = TypedDict(
+    "EventMessageType",
+    {
+        "version": str,
+        "id": str,
+        "detail-type": Required[Union[str, List[str]]],
+        "source": Required[Union[str, List[str]]],
```

Comment:
These two should also be quoted, that's why the linter is failing

Reply:
Oh yes, thanks!